fmt/printf.ts

```diff
@@ -47,6 +47,7 @@
  * | `T`   | type of arg, as returned by `typeof`                           |
  * | `v`   | value of argument in 'default' format (see below)              |
  * | `j`   | argument as formatted by `JSON.stringify`                      |
+ * | `i I` | argument as formatted by `Deno.inspect`                        |
```

Comment:
Let's document the difference of `i` and `I`

```suggestion
 * | `i`   | argument as formatted by `Deno.inspect`                        |
 * | `I`   | argument as formatted by `Deno.inspect` in compact format      |
```